src/core/componentSystems/MaterialStyleSystem.js

```diff
@@ -36,6 +36,9 @@ export class MaterialStyleSystem extends MRSystem {
             if (entity instanceof MRDivEntity) {
                 this.setBackground(entity);
             }
+            if (entity instanceof MRVideo && entity.playing) {
+                entity.texture.needsUpdate = true;
+            }
```

Comment:
Is a texture of `MRVideo` guaranteed to be as `VideoTexture`? If so we may not need to manually call `texture.needsUpdate = true` because it's automatically (and more efficiently with [requestVideoFrameCallback](https://developer.mozilla.org/en-US/docs/Web/API/HTMLVideoElement/requestVideoFrameCallback))

https://threejs.org/docs/#api/en/textures/VideoTexture.needsUpdate

Even if it isn't guaranteed, we may add a guard like

```
if (entity instanceof MRVideo && entity.playing && !entity.texture.isVideoTexture) {
  entity.texture.needsUpdate = true;
}
```

Reply:
good to know, since it's guaranteed, i'll remove it ~

Reply:
√